docs/references/nextjs/auth.mdx

```diff
@@ -11,7 +11,40 @@ The `auth()` helper does require [Middleware](/docs/references/nextjs/auth-middl
 
 A Route Handler added to [`publicRoutes`](/docs/references/nextjs/auth-middleware#making-pages-public-using-public-routes) can still use the [`auth()`](/docs/references/nextjs/auth) helper to return information about the user or their authentication state, or to control access to some or all of the Route Handler.
 
-## Retrieving userId
```

Comment:
Do we need a hash redirect? (Maybe you've already done it elsewhere that I haven't noticed)

Reply:
I scanned the docs for this hash, and its not referenced anywhere. other than that, we haven't done hash redirects in the past.